outputs/BaseOutput.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace GameOfLife\outputs;
+
+use GameOfLife\Field;
+use Ulrichsg\Getopt;
+
+/**
+ * The Base-Output
```

Comment:
Remember the summary sentence of a class must end with a .
Example: "Base-Output for all outputs of GameOfLive."